snap/README.md

```diff
@@ -534,61 +559,118 @@ root@edgex:~# cd edgex-go && snapcraft --destructive-mode
+
+4. Change your code, then repeat step 2 and 3
+
+5. Ship the tested snap file
 
 ```bash
-$ snapcraft try # produces prime dir instead of *.snap file
-...
-You can now run `snap try /home/ubuntu/go/src/github.com/edgexfoundry/edgex-go/prime`.
-$ sudo snap try --devmode prime # snap try works the same as snap install, but expects a directory
-edgexfoundry 1.0.0-20190513+0620a8d1 mounted from /home/ubuntu/go/src/github.com/edgexfoundry/edgex-go/prime
-$
+sudo snap pack ./prime
 ```
 
 #### Interfaces
 
-After installing the snap, you will need to connect interfaces and restart the snap. The snap needs the `hardware-observe`, `mount-observe`, and `system-observe` interfaces connected. These are automatically connected using snap store assertions when installing from the store, but when developing the snap and installing a revision locally, use the following commands to connect the interfaces:
+After installing the snap, you will need to connect interfaces and restart the snap. 
+Interfaces are automatically connected using snap store assertions when installing from the store, 
+but when developing the snap and installing a revision locally, use the commands in this section to connect the interfaces.
+
+To see which interfaces the snap is using, and which interfaces it could use but isn’t:
+
+```bash
+sudo snap connections edgexfoundry
+```
+
+```
+Interface                         Plug                                  Slot                                                    Notes
+content[edgex-secretstore-token]  edgexfoundry:edgex-secretstore-token  edgex-app-service-configurable:edgex-secretstore-token  -
+content[edgex-secretstore-token]  edgexfoundry:edgex-secretstore-token  edgex-device-mqtt:edgex-secretstore-token               -
+content[edgex-secretstore-token]  edgexfoundry:edgex-secretstore-token  edgex-device-snmp:edgex-secretstore-token               -
+home                              edgexfoundry:home                     :home                                                   -
+network                           edgexfoundry:network                  :network                                                -
+network-bind                      edgexfoundry:network-bind             :network-bind                                           -
+removable-media                   edgexfoundry:removable-media          -                                                       -
+```
+
+Manual connections:
+
+```bash
+snap connect <snap>:<plug interface> <snap>:<slot interface>
+```
+
+Here is an example:
 
 ```bash
-$ sudo snap connect edgexfoundry:hardware-observe
-$ sudo snap connect edgexfoundry:system-observe
-$ sudo snap connect edgexfoundry:mount-observe
+sudo snap connect edgexfoundry:edgex-secretstore-token <your-snap>:<slot>
 ```
 
 After connecting these restart the services in the snap with:
 
 ```bash
-$ sudo snap restart edgexfoundry
+sudo snap restart edgexfoundry
```

Comment:
I don't think this is correct - after we connect the edgex-secretstore-token plug/slot from edgexfoundry to another snap, there shouldn't be any need to restart edgexfoundry?

Reply:
You are right, I just tested it. Removed.